dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -2583,7 +2586,7 @@
                   "type": "object",
                   "properties": {
                     "name": {
-                      "$ref": "#/definitions/fullName"
+                      "type": "string"
```

Comment:
Does this need a unit test? I don't see a test validating `sources` here :https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/test/schemas/28-8832/schema.spec.js